modules/cli-options/src/main/scala/scala/cli/commands/SpecificationLevel.scala

```diff
@@ -0,0 +1,40 @@
+package scala.cli.commands
+
+import caseapp.annotation.Tag
+import os.copy
+
+sealed trait SpecificationLevel {
```

Comment:
NIT Make that extend `Product with Serializable`, so that the sequences below are typed as just `Seq[SpecificationLevel]`?